### PR TITLE
III-3570 Add endpoint to update subEvents status(es) for single and multiple calendar types

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added

- Added new endpoint to update the status of subEvents for events with single or multiple calendar type
- Added support for new status projection on events and places in tests

---
Ticket: https://jira.uitdatabank.be/browse/III-3570

### CODE REVIEW

**Comment by jdrieghe** [comment]
@bertramakers we still need to fix the tests that were broken in this library by updating `udb3-php` with the latest changes

File: tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php
Comment (bertramakers) [inline tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php:556]: Hmm I think we might need to omit this if it's empty, instead of setting it to an empty array/object 🤔

Comment (bertramakers) [reply tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php:556]: I've omitted it as part of https://github.com/cultuurnet/udb3-php/pull/482

**Review by bertramakers — APPROVED** [review]
@jdrieghe Since you're the author of this PR and cannot approve I've gone ahead and marked this as approved. But can you go over my extra commits before we merge this? Especially since it's now more commits by me than you 😅 Or alternatively @LucWollants could go over everything.

It will also require a merge of https://github.com/cultuurnet/udb3-php/pull/481 first which will require an approval from you (or @LucWollants of course). And then we'll have to update the composer dependency in this PR again.

File: src/Http/Event/UpdateSubEventsStatusRequestHandler.php
Comment (jdrieghe) [inline src/Http/Event/UpdateSubEventsStatusRequestHandler.php:36]: It was worth a try :D

**Comment by jdrieghe** [comment]
Thanks for the follow-up work, @bertramakers.
This looks good and can be merged, I think.

File: src/Http/Event/UpdateSubEventsStatusRequestHandler.php
Comment (bertramakers) [reply src/Http/Event/UpdateSubEventsStatusRequestHandler.php:36]: Yeah I'd also have preferred this since it was a lot cleaner in the controller provider in Silex, but I got an error along the lines of `0 arguments passed to UpdateSubEventsStatusRequestHandler, 2 expected`. So I think Silex was trying to instantiate the class itself and failing to do so correctly.

**Pull request merged**
